sunbeam-python/sunbeam/commands/prepare_node.py

```diff
@@ -96,6 +96,32 @@
 # This also gives access to the ssh binary to the snap.
 sudo snap connect openstack:ssh-keys
 
+# Check Juju is already installed with correct version
+if [ -n "$(command -v juju)" ]; then
```

Comment:
Do we really need to check with the juju version, can we just check the juju channel instead.

Comparing `snap list | grep juju | awk '{print $4}'`  with JUJU_CHANNEL